test/jerboa/format/head_test.exs

```diff
@@ -105,35 +105,29 @@ defmodule Jerboa.Format.HeaderTest do
     end
   end
 
-  describe "Header.*.encode/1" do
+  test "Header.Length.encode/1 encodes length into 16 bits (two bytes)" do
+    x = Header.Length.encode(%Params{body: <<0,1,0,1>>})
 
-    test "bind request method and class in 14 bit type" do
-      x = Header.Type.encode(%Params{class: :request, method: :binding})
-
-      assert 14 === bit_size x
-      assert <<0x0001::16>> == <<0::2, x::14-bits>>
-    end
-
-    test "length into 16 bits (two bytes)" do
-      x = Header.Length.encode(%Params{body: <<0,1,0,1>>})
-
-      assert 16 === bit_size x
-      assert <<0, 4>> = x
-    end
+    assert 16 === bit_size x
+    assert <<0, 4>> = x
```

Comment:
I think it would be better to have an assertion like `<<4 :: size(16)>> = x`. It captures both, doesn't it?